lib/lib_ssl/tls_mini/src/StackThunk_light.cpp

```diff
@@ -27,7 +27,7 @@
 
 #include "tasmota_options.h"
 
-#if defined(ESP8266) && defined(USE_TLS)
+#if defined(ESP8266) && (defined(USE_TLS) || defined(USE_SENDMAIL))
```

Comment:
Isn't this solved with the latest re-organization of config header files?

Reply:
yes it should but at least it does not compile (sometimes) strange ?  platformio bug?
ok, reverted this as it SHOULD work

Reply:
May be tasmota_configurations.h is missing something like ?
```
#ifdef USE_SENDMAIL
#define USE_TLS
#endif
```

Reply:
this should set TLS but obviously doesn't do it (always)

myuserconfig =>

#if defined(USE_MQTT_TLS) || defined(USE_SENDMAIL) || defined(USE_TELEGRAM) || defined(USE_WEBCLIENT_HTTPS) || defined(USE_ALEXA_AVS)
  #define USE_TLS                                  // flag indicates we need to include TLS code
#endif

Reply:
THis should be at the end of tasmota_configurations
The compiler will not see that is USE_SENDMAIL is added in user_config_override.h as the order is
- my_user_config
- user_config_override
- tasmota_configurations

Fixing the dependencies should be done in the latest

Reply:
i have a slightly outdated version. this must have been corrected very recently
with the updated version it works now